src/cdk-experimental/combobox/combobox.ts

```diff
@@ -189,22 +211,29 @@ export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
     return this.disabled ? null : '0';
   }
 
-  private _setComboboxValue(value: T) {
+  private _setComboboxValue(value: T | T[]) {
 
     const valueChanged = (this.value !== value);
     this.value = value;
 
     if (valueChanged) {
-      this.panelValueChanged.emit(value);
-      this._setTextContent(value);
+      this.panelValueChanged.emit(coerceArray(value));
+      if (this._autoSetText) {
+        this._setTextContent(value);
+      }
     }
   }
 
-  private _setTextContent(content: T) {
+  private _setTextContent(content: T | T[]) {
     const contentArray = coerceArray(content);
     this._elementRef.nativeElement.textContent = contentArray.join(' ');
   }
 
+  private _isTextTrigger() {
+    const tagName = this._elementRef.nativeElement.tagName.toLowerCase();
```

Comment:
I would add a TODO to also test for `contenteditable` (I think supporting that will take some more thought before doing it)

Reply:
Sounds good.